src/command.js

```diff
@@ -1,43 +1,58 @@
 import { CURRENT_STABLE } from './config.js';
 
 export class Command {
-    command                         = '';
-    php                             = CURRENT_STABLE;
-    extensions                      = [];
-    ini                             = [];
-    dependencies                    = 'locked';
-    ignore_php_platform_requirement = false;
-    additional_composer_arguments   = [];
+    command = '';
+
+    php = CURRENT_STABLE;
+
+    extensions = [];
+
+    ini = [];
+
+    dependencies = 'locked';
+
+    ignorePhpPlatformRequirement = false;
```

Comment:
Aren't these changes potential BC breaks?

Reply:
I'd say no as we do not provide an API here, right?
Only changing application code. Input configs and output matrix havent changed.

Reply:
Isn't this serialized and given to the next action?

Reply:
Nope, there is a dedicated method `toJSON` for that.